From 206c9b97c2ff7ebacc12cd0ccd85eb9a6fbad22f Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Fri, 26 Oct 2012 16:10:38 +0800
Subject: [PATCH 2189/2463] ENGR00231321-1 mx6: remove wm8958 in
 imx6_defconfig

not set wm8958 codec as builtin in imx6_defconfig

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index f1bab5f..f62ea0e 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1848,7 +1848,7 @@ CONFIG_SND_IMX_SOC=y
 CONFIG_SND_MXC_SOC_MX2=y
 CONFIG_SND_MXC_SOC_SPDIF_DAI=y
 CONFIG_SND_SOC_IMX_SGTL5000=y
-CONFIG_SND_SOC_IMX_WM8958=y
+# CONFIG_SND_SOC_IMX_WM8958 is not set
 CONFIG_SND_SOC_IMX_WM8962=y
 CONFIG_SND_SOC_IMX_CS42888=y
 # CONFIG_SND_SOC_IMX_SI4763 is not set
@@ -1856,13 +1856,13 @@ CONFIG_SND_SOC_IMX_SPDIF=y
 CONFIG_SND_SOC_IMX_HDMI=y
 CONFIG_SND_SOC_I2C_AND_SPI=y
 # CONFIG_SND_SOC_ALL_CODECS is not set
-CONFIG_SND_SOC_WM_HUBS=y
+# CONFIG_SND_SOC_WM_HUBS is not set
 CONFIG_SND_SOC_MXC_HDMI=y
 CONFIG_SND_SOC_MXC_SPDIF=y
 CONFIG_SND_SOC_SGTL5000=y
 CONFIG_SND_SOC_CS42888=y
 CONFIG_SND_SOC_WM8962=y
-CONFIG_SND_SOC_WM8994=y
+# CONFIG_SND_SOC_WM8994 is not set
 # CONFIG_SOUND_PRIME is not set
 CONFIG_AC97_BUS=y
 CONFIG_HID_SUPPORT=y
-- 
1.7.7.4

